src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskCancelByIdControllerTest.java

```diff
@@ -282,5 +293,36 @@ public void user_should_cancel_task_when_grant_type_challenged_and_permissions_r
         common.cleanUpTask(taskId);
     }
 
+    @Test
+    public void user_should_cancel_task_when_granular_permission_satisfied() {
```

Comment:
can we update the test name so its clear which AC/scenario its satisfying? This one appears to be for CANCELOWN.  Maybe "should_successfully_cancel_task_when_granular_permission_cancel_own_satiisfed"

Reply:
Name changed to user_should_cancel_task_when_granular_permission_cancel_own_satisfied()